src/internal.rs

```diff
@@ -665,7 +665,7 @@ pub(crate) struct Rav1dTaskContext_task_thread {
 
 #[repr(C)]
 pub(crate) struct Rav1dTaskContext {
-    pub f: *const Rav1dFrameContext,
+    pub f: *mut Rav1dFrameContext,
```

Comment:
in practice mutation happens through this pointer. I'm assuming c2rust makes a best effort when assigning const/mut to a pointer and it just couldn't figure it out in this case?

Reply:
`c2rust transpile` just copies how C declared things, so this must've been declared as a `const Dav1dFrameContext*` in C.  `const` and `mut` don't really matter for raw pointers in Rust since you can cast between them like in C.  I think it only matters if it originally comes from a `const` or `mut` source.